### PR TITLE
Adds robust import of OMX for matrix and index names that are too long

### PR DESCRIPTION
**Pull request by pedrocamargo**

### CODE REVIEW

File: aequilibrae/matrix/aequilibrae_matrix.py [hunk 1]
```diff
@@ -248,20 +271,33 @@ def create_from_omx(
             raise ValueError("AequilibraE only supports square matrices")
         zones = shp[0]
 
+        if robust:
+            core_names = []
+            for c in do_cores:
```
Comment (jamiecook) [inline aequilibrae/matrix/aequilibrae_matrix.py]: Can't this just be a comprehension like the else case?

Comment (pedrocamargo) [reply aequilibrae/matrix/aequilibrae_matrix.py]: No.  core_names is an input to the computation of its next element

Comment (jamiecook) [reply aequilibrae/matrix/aequilibrae_matrix.py]: ah

```python
core_names = functools.reduce(lambda acc,n: robust_name(n, CORE_NAME_MAX_LENGTH, acc),do_cores, [])
idx_names = functools.reduce(lambda acc,n: robust_name(n, INDEX_NAME_MAX_LENGTH, acc),do_idx, [])
```

Comment (pedrocamargo) [reply aequilibrae/matrix/aequilibrae_matrix.py]: Incorporated!  Thanks!

**Pull request merged**